packages/storefront-lwc/src/modules/commerce/errorPage/errorPage.html

```diff
@@ -0,0 +1,85 @@
+<!--
+    Copyright (c) 2020, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+<template>
+    <commerce-context-heading if:true={pagenotfound} header-text="Page not found"></commerce-context-heading>
+    <commerce-context-heading if:true={servererror} header-text="Something went wrong"></commerce-context-heading>
+    <div class="container" if:true={hasError}>
+        <div if:true={devMode}>
+            <div if:true={error} class="row pt-2">
+                <pre>
+                    <code>
+                        <template for:each={error} for:item="errorLine">
+                            <span key={errorLine.index}>{errorLine.line}</span>
+                        </template>
+                    </code>
+                </pre>
+            </div>
+            <div if:true={stack} class="row pt-2">
+                <pre>
+                    <code>
+                        <template for:each={stack} for:item="stackLine">
+                            <span key={stackLine.index}>{stackLine.line}</span>
+                        </template>
+                    </code>
+                </pre>
+            </div>
+            <div if:false={error} class="row pt-4">
+                <div class="col"></div>
+                <div class="col-12 col-sm-6" if:true={pagenotfound}>
+                    <p class="text-left">We can't find the page you're looking for!</p>
+                    <p class="text-left">You can either visit home page or contact our support team.</p>
+                </div>
+                <div class="col-12 col-sm-6" if:true={servererror}>
+                    <p class="text-left">Sorry, something went wrong. Please try again or contact us.</p>
+                </div>
+                <div class="col"></div>
+            </div>
+            <div class="row pt-2">
+                <div class="col"></div>
+                <div class="col-12 col-sm-6 text-center">
+                    <p>
+                        <lwce-link href="/" class="btn btn-primary btn-block"> Homepage </lwce-link>
+                    </p>
+                </div>
+                <div class="col"></div>
```

Comment:
Why do we need this empty `div`? Actually there are a few empty `div`s L31, L39, L42, L48, L52, L60, L63, L74. We should consider removing them.

Reply:
We need it to centralize the middle content div to the page. The same is followed in [basket.html](https://github.com/SalesforceCommerceCloud/sfcc-sample-apps/blob/4dc65eb51e34cd1028095cb810bd0272188137d3/packages/storefront-lwc/src/modules/commerce/basket/basket.html#L33)

Reply:
You can remove the the empty `div`s and put `justify-content-center` on the row. You might need to make the content div col-6 as well

Reply:
Done for both errorPage and basket.